src/NServiceBus.Core/Transports/Msmq/QueuePermissions.cs

```diff
@@ -14,6 +14,11 @@ public static void CheckQueue(string address)
             var queuePath = msmqAddress.PathWithoutPrefix;
 
             Logger.Debug($"Checking if queue exists: {queuePath}.");
+            if (msmqAddress.IsRemote)
+            {
+                Logger.Info($"This endpoint cannot verify the existence of the remote queue '{queuePath}'. Make sure the queue exists, and that the address is correct. Messages can potentially end up in the dead letter queue if configured incorrectly.");
```

Comment:
I get the impression that there is something with this endpoint that results in us not being able to check? 

Should we state that "Since queue is remote we can't check permissions on it" or similar?